csp_adapter_symphony/adapter.py

```diff
@@ -337,109 +416,49 @@ def stop(self):
         if self._running:
             # in order to unblock current requests.get, send a message to one of the rooms we are listening on
             self._running = False
-            self._delete_datafeed_if_set()
-            if self._exit_msg:
-                send_symphony_message(self._exit_msg, next(iter(self._room_ids)), self._message_create_url, self._header)
+            try:
+                self._delete_datafeed_if_set()
+                if self._exit_msg:
+                    send_symphony_message(self._exit_msg, next(iter(self._room_ids)), self._message_create_url, self._header)
+            except Exception:
+                log.error("Error on sending exit message and deleting datafeed on shutdown", exc_info=True)
```

Comment:
can do `log.exception` instead of `log.error(..., exc_info=True)`

Reply:
oh didn't know, thanks!